src/test/scala/outwatch/DomEventSpec.scala

```diff
@@ -467,7 +467,7 @@ class DomEventSpec extends JSDomSpec {
         doubleStream <- Handler.create[Double]
         boolStream <- Handler.create[Boolean]
         htmlElementStream <- Handler.create[html.Element]
-        svgElementTupleStream <- Handler.create[(svg.Element, svg.Element)]
```

Comment:
What is the reason to use `Element` instead of `svg.Element`?

Reply:
Oh good catch. I think this was just a name clash between `outwatch.dom.dsl.svg` and `org.scalajs.dom.svg`. So we should keep svg.Element here.